File: abc/abc346/a.cs
// https://atcoder.jp/contests/abc346/submissions/51670669

using static System.Console;

public class A
{
  static void Main()
  {
    int n = int.Parse(ReadLine());
    string[] a = ReadLine().Split();
    for(int i = 1; i < n; i++)
    {
      Write($"{int.Parse(a[i - 1]) * int.Parse(a[i])} ");
    }
  }
}

File: abc/abc346/a.cpp
// https://atcoder.jp/contests/abc346/submissions/51670578

#include <bits/stdc++.h>
using namespace std;

int main() {
  int n;
  cin >> n;

  deque<int> a(n);
  for(auto &z : a) cin >> z;

  while(a.size() >= 2) {
    cout << a[0] * a[1] << " ";
    a.pop_front();
  }
}
